test/e2e/scenarios/ACL/acl_create_update.yaml

```diff
@@ -74,12 +74,15 @@ steps:
       spec:
         userNames:
           - userone$RANDOM_SUFFIX
+        tags:
+          - key: "test_key_1"
+            value: "test_value_1"
     wait:
       status:
         conditions:
           ACK.ResourceSynced:
             status: "True"
-            timeout: 180
+            timeout: 300
```

Comment:
No way it takes 5 minutes to update tags. That's insane!

Reply:
With 180 second it failed. I checked controller logs, and logs show updating tags all the time (with different orders) so `ResourceSynced` cannot be true. Should we pause this PR and wait for new tag compare function being added to `code-generator` and `runtime`?

Reply:
I've tried locally. It really took a long time to finish update. More than 5 minutes. Really think we need to wait for new version of code-generator/runtime that includes new compare tags function/method. Extra tags update calls slow everything down.